libsolidity/formal/CVC4Interface.cpp

```diff
@@ -60,17 +60,21 @@ void CVC4Interface::addAssertion(Expression const& _expr)
 	{
 		m_solver.assertFormula(toCVC4Expr(_expr));
 	}
-	catch (CVC4::TypeCheckingException const&)
+	catch (CVC4::TypeCheckingException const& _e)
 	{
-		solAssert(false, "");
+		solAssert(false, _e.what());
```

Comment:
Can this be nullptr? What happens in that case?

Reply:
I think ``what`` in an exception is required to always be a valid null-terminated string - so if it returns nullptr, that'd be a bug in CVC4.